README.md

```diff
@@ -31,13 +31,24 @@ if it stops at the wrong time (the window is short), it can disable your partiti
 4. Go to `http://4.3.3.4/` in your browser. This will take you to the main menu.
 5. Click `Configure Wifi` and set up your actual wifi. The device will reboot.
 6. Find the IP of the device, and connect to it.
-7. Click `Read partitions` to see the current partition table.
+7. Click `List partitions` to see the current partition table.
 This should look reasonable before proceeding, otherwise it might not be able to read your device's partition table.
-It will also tell you if you're in the right app partition.
+It will also tell you if you're in the right app partition. Ex:
+```
+Partition:
+Addr: 0x00008000
+Type: 01, Subtype: 02, Addr: 0x00009000, Size: 0x00005000 (20K), Label: nvs
+Type: 01, Subtype: 00, Addr: 0x0000e000, Size: 0x00002000 (8K), Label: otadata
+Type: 00, Subtype: 10, Addr: 0x00010000, Size: 0x00140000 (1280K), Label: app0
+Type: 00, Subtype: 11, Addr: 0x00150000, Size: 0x00140000 (1280K), Label: app1
+Type: 01, Subtype: 82, Addr: 0x00290000, Size: 0x00170000 (1472K), Label: spiffs
+Running: Addr: 0x00010000, Label: app0
+Next:    Addr: 0x00150000, Label: app1
+```
 8. Click `Fix partitions`, and await the results. If it says that it's not on the first partition, upload this firmware again, and try again.
 9. If you're ok, it'll say ready on the bottom and reboot.
-10. Reconnect to `EPM-AP`, set your wifi again, reboot, find the device and connect to it.
-11. Check `Read partitions` to see if it worked.
+10. Reload the page once reboot is complete.
```

Comment:
After the partition fix and reboot the device connected to the WiFi i had configured in step (5); it didn't start its own AP.

Reply:
Thanks!